IS7024Project/Crime.cs

```diff
@@ -6,7 +6,7 @@
 //
 //    var crime = Crime.FromJson(jsonString);
 
-namespace QuickType
+namespace CrimeSpace
```

Comment:
Changed the Quicktype as CrimeSpace to be more descriptive.

Reply:
A better namespace, yes.
You can specify the namespace in QuickType as well... I figured this out after I recorded the video.  :)